keyboards/dz60/keymaps/marianas/keymap.c

```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT(
```

Comment:
For your reference, it may be a good idea to number/label the layers. 

You can see what I mean in the planck and clueboard's default keymaps: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c
https://github.com/qmk/qmk_firmware/blob/master/keyboards/clueboard/66/keymaps/default/keymap.c

This isn't necessary, but may help to keep it more readable for you.